ui/view/game.py

```diff
@@ -260,6 +263,18 @@ def update(self):
         """
         self._board_view.update()
 
+    def update_timer(self, time: timedelta):
+        """
+        Updates the game view timer.
+        :param time: a time to display.
+        """
+        time_string = str(time)
+        minutes, seconds = time_string.split(":")[1:]
+        seconds_string = F"{int(float(seconds))}".zfill(2)
+        milliseconds_string = F"{int(time.microseconds / pow(10, 3))}".zfill(3)
+        self._timer_text.set(F"{minutes}:{seconds_string}.{milliseconds_string}")
+
```

Comment:
Since the timer would be updated so much by itself, I didn't want to invoke updates all the time for the whole view, so I just made it into an isolated function that can be called independently.